src/app/components/home-intro/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Scroll from 'react-scroll'; /* Animate and scroll to location in document */
+import transitionOnScroll from 'app/lib/transition-on-scroll';
+
+import SVG from 'app/components/svg';
+import Video from 'app/components/video';
+
+// const rainbowColours = ['#ED0082', '#E60C29', '#FF5519', '#FFBF02', '#96CC29', '#14C04D', '#16D6D9', '#009CF3', '#143FCC', '#6114CC', '#111111'];
+const colours = ['#000000', '#FFFFFF'];
+
+function HomeIntro({ scrollProgress, mousePosition, scrolling, loaded }) {
+
+  /*
+    Create stack of logos in a sort of 3D space that move dependant on the mouse position.
+  */
+  const logoLayers = colours.map((fill, i) => {
+    let modifier = 3 * ((colours.length - i) * (colours.length - i) * 0.9);
+    let modifierRotate = 2 * ((colours.length - (i-1)) * (colours.length - (i-1)));
```

Comment:
👍 I'm not. Ethel suggested it too. I will aim to implement one. The lack of spaces was a weird relapse from me - subconscious trying to make shorter lines!